src/v2/stores/networkOverview.js

```diff
@@ -1,29 +1,114 @@
-import {configure, flow, observable, action} from 'mobx';
+import {
+  configure,
+  flow,
+  observable,
+  action,
+  decorate,
+  observe,
+  computed,
+} from 'mobx';
+import {parse, format} from 'date-fns';
+import {
+  map,
+  keys,
+  compose,
+  pick,
+  mapValues,
+  toPairs,
+  find,
+  includes,
+  identity,
+  pickBy,
+} from 'lodash/fp';
+import {camelizeKeys} from 'humps';
 import * as API from 'v2/api/stats';
+import {parseBlock, parseTransaction} from 'v2/utils/parseMessage';
 
-configure({enforceActions: 'always'});
-
-const OverviewStore = observable(
-  {
-    state: 'pending',
-    globalStats: {},
-
-    getStats: flow(function* getStats() {
-      this.state = 'loading';
-      try {
-        const res = yield API.getStats();
-        this.state = 'loaded';
-        this.globalStats = res.data;
-      } catch (e) {
-        this.state = 'error';
+import calcChanges from '../utils/calcChanges';
+
+configure({enforceActions: 'observed'});
+
+class OverviewStore {
+  globalStats = {};
+  statsChanges = {};
+  txnStats = {};
+
+  constructor() {
+    observe(this, 'globalStats', ({oldValue, newValue}) => {
+      if (!keys(oldValue).length) {
+        return;
       }
-    }),
-  },
-  {
-    state: observable,
-    globalStats: observable,
-    getStats: action.bound,
-  },
-);
-
-export default OverviewStore;
+
+      this.statsChanges = compose(
```

Comment:
I think this should probably be removed until the changes are calculated correctly (daily etc. vs per-message)